osu.Game/Database/RealmLive.cs

```diff
@@ -47,7 +55,10 @@ public void PerformRead(Action<T> perform)
                 return;
             }
 
-            using (var realm = Realm.GetInstance(data.Realm.Config))
+            if (realmFactory == null)
+                throw new ArgumentException(@"Realm factory must be provided for a managed instance", nameof(realmFactory));
```

Comment:
Not sure why these cases are null-checked explicitly, contrary to `PerformWrite()` which just uses a null-forgiving operator. This null case should be impossible, as the realm factory is guaranteed to be present by the ctor if the object is managed.

Reply:
No longer exists.